README.md

```diff
@@ -26,4 +26,12 @@ It consists of two apps:
  
 **Limitations**
 - The ConnectIQ app is optimized for Garmin FR 23x, but also runnable on FR 245, FR 6xx, FR 735xt, vivoactive 3&4, Venu
-- Potential issue with Garmin's Android BLE library on Android 11 devices
\ No newline at end of file
+- Potential issue with Garmin's Android BLE library on Android 11 devices
+
+**alsama/wormnav CHANGES:**
```

Comment:
This comments in Readme.md could be changed to simply tell that now app is multi language.